### DISCUSSION TITLE
More of a question than an Issue. :)

### DISCUSSION THREAD

**Discussion by BobRathke** [hassio-addons/addon-wireguard#78] [Q&A]
Thanks for all your wonderful HA integrations & add ons. I really appreciate your work and plan on donating.

I would like to use Wire Guard to access more than just the HA server. I would also like to access my local 192.X.X.X network.

I am running HASS.IO with docker. I was able to install wireguard and get a VPN connection up and running. I am confused on how to add the ability to jump from the 172.X network over to the 192.X network.

Thanks.

**Comment by frenck** [comment]
It should just work from the default settings, as the add-on routes the traffic out.

**Comment by BobRathke** [comment]
Hi Franck - I am unable to ping any of my 192.X network after installing.
________________________________

**Reply by frenck** [reply]
It is a normal routing when using the default settings... I dunno what to say 🤷‍♂️
